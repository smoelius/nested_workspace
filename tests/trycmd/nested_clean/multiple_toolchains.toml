args = ["nested", "clean"]

[bin]
name = "cargo-nested"

[fs]
cwd = "../../../fixtures/multiple_toolchains"
