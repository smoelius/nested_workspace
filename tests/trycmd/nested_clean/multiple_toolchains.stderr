     Removed [..] files[..]
<<< [CWD]/beta
     Removed [..] files[..]
>>> [CWD]/beta
<<< [CWD]/nightly
     Removed [..] files[..]
>>> [CWD]/nightly
